smrr_zed_scanner_node:
  ros__parameters:
    point_cloud_topic: "/zed/zed_node/point_cloud/cloud_registered"
    camera_z_offset: 1.25 #1.3
    transform_tolerance: 0.01
    ground_level: -0.2  #w.r.t. base_footprint
    angle_min: -3.1415927410125732 #-1.5708,  # -M_PI/2
    angle_max: 3.1415927410125732 #1.5708,  # M_PI/2
    angle_increment: 0.017501909285783768  #0.0087,  #S M_PI/360.0
    scan_time: 0.3333
    activate_cliff_detector: true
    activate_filter: true
    cliff_range_min: 0.35
    cliff_range_max: 5.0
    obstacle_range_min: 0.35
    obstacle_range_max: 4.0
    obstacle_height_min: 0.5
    obstacle_height_max: 2.0  
    use_inf: True
    inf_epsilon: 1.0


